test/integration/suites/upgrade/versions.txt

```diff
@@ -4,3 +4,4 @@
 1.1.3
```

Comment:
I think we should remove the 1.1.x series at this point?

Reply:
Yeah good point, looks like this got missed in the past

Reply:
We remove the previous minor release when we're the next release is advancing the minor version. So having 1.1.x in here was expected during the 1.2.x release cycle, since we wanted to ensure we could move from any 1.1.x build to any 1.2.x build.